v3/src/data-interactive/handlers/data-context-handler.ts

```diff
@@ -88,7 +88,11 @@ export const diDataContextHandler: DIHandler = {
 
         if (managingController) {
           const tile = findTileFromNameOrId(managingController)
-          if (tile) dataContext.setManagingControllerId(tile.id)
+          if (tile) {
+            dataContext.setManagingControllerId(tile.id)
+          } else {
+            dataContext.setManagingControllerId("")
+          }
```

Comment:
Could also do:
```suggestion
          dataContext.setManagingControllerId(tile?.id ?? "")
```
or just
```typescript
          dataContext.setManagingControllerId(tile?.id)
```
if `undefined` were interpreted as the empty string.

Reply:
Much cleaner, thanks for the suggestion 👍